les/utils/expiredvalue.go

```diff
@@ -92,10 +93,13 @@ func (e *ExpiredValue) Add(amount int64, logOffset Fixed64) int64 {
 		e.Exp = integer
 	}
 	if base >= 0 || uint64(-base) <= e.Base {
+		fmt.Println("base=", base, uint64(-base), e.Base, uint64(base))
 		e.Base += uint64(base)
```

Comment:
This is the line that is causing the problem. On arm64, `uint64(base) == 0` whereas on amd64 it's basically -1. I am trying to determine if the problem is with golang or if it's with arm64.